### ISSUE TITLE
File Page needs a Term Tab

### ISSUE THREAD

**Issue by shlake** [IQSS/dataverse#4391]
Now that there is a separate files page & in a future release of dataverse each file is going to have its own DOI, the files page needs a Terms Tab.

Main reason for this is to give more information for Restricted files. 

It is sort of related to this issue: https://github.com/IQSS/dataverse/issues/3997
which talks about needing more restricted file information, such that a log in is needed.

Maybe a Terms tab on the Files page would be a resolution to issue #3997 ?

**Comment by mheppler** [comment]
This issue seems like the most logical place to drop a reminder to look into improving the performance of the file pg, as noted in _File page - slow/excessive queries (regressive*) #3960_. Before we add more features to this page, we should review and refactor as much of the existing code as possible to make sure we are not adding excessive burden to a page that is already under performing.

**Comment by TaniaSchlatter** [comment]
Sonia also has use cases for this feature. Some Murray datasets have different terms of use for different files in a dataset.

**Comment by TaniaSchlatter** [comment]
@shlake @adam3smith, our understanding after discussion is that this issue is about displaying terms that apply to a file on the file page. There is no new functionality requested. Is this correct? This issue suggests having the info in a Terms tab on the file page, and @adam3smith requested having it on the metadata tab in [#3997](https://github.com/IQSS/dataverse/issues/3997).

**Comment by shlake** [comment]
@TaniaSchlatter 

What I think I really wanted when I opened this issue was to have the ability (with a "term tab" on the file page) to be able to set different terms of use per file in the same dataset. 

This is what I would like to see, and I thought there was an issue about this: to have an option to set terms/license per file OR at dataset level, but I can't seem to find it now.

But if this issue is about showing the file "terms" of use on the file page - it doesn't matter if it is a separate tab (which the way it reads is this issue) or in the file metadata #3997

**Comment by TaniaSchlatter** [comment]
Thanks for clarifying, @shlake. The ability to define terms for files will happen with the DataTags integration #871.

**Comment by adam3smith** [comment]
I also want per-file terms, but the linked issue I opened was indeed just for displaying terms on the file landing page, no additional functionality. That would already help us plenty.

**Comment by philippconzett** [comment]
I'd like to revive the discussion about file-level license support. I think the topic was first introduced in [this Google Group discussion thread](https://groups.google.com/u/1/g/dataverse-community/c/1hV5_kewVpE/) and has since been discussed in different settings.

Together with multiple license support (see [current support](https://guides.dataverse.org/en/6.3/installation/config.html#configuring-licenses) and [issue](https://github.com/IQSS/dataverse/issues/8512)), file-level license support is crucial for a repository to align with the FAIR principles.

Considering how licensing information is provided to DataCite (see, e.g., [this question in the PID Forum](https://pidforum.org/t/publishing-datasets-with-multiple-licenses/2460)) and other relevant services, file-level license support is related to the way file-level metadata information is coded; see #5086.

A possible implementation could thus look like this:
- Each file in a dataset needs to be assigned a standard license or customized terms of use.
- The dataset itself or the metadata record describing the entire dataset - if you want - is assigned the CC0 waiver accompanied by the community norm reminder to always cite your sources.

**Comment by LauraHuisintveld** [comment]
@philippconzett and others:  We are also very interested in having licence and access information at file level. This information should be machine-readable, so I agree that having this in the metadata-export(s) is crucial. We (DANS) are participating in a national project, in which we could probably try to spend some time to take a look at this.

**Comment by cmbz** [comment]
2024/09/09: Keeping, still high level of interest.

**Comment by LauraHuisintveld** [comment]
> * The dataset itself or the metadata record describing the entire dataset - if you want - is assigned the CC0 waiver accompanied by the community norm reminder to always cite your sources. See [Metadata licensing #6888](https://github.com/IQSS/dataverse/issues/6888).

I like this idea, but I am wondering if services like OpenAIRE will understand that the licence at dataset-level is about the metadata only?
https://guidelines.openaire.eu/en/latest/data/use_of_datacite.html#accessrights

**Comment by pdurbin** [comment]
There's a new "Adding Multiple Licenses to a Dataset in Dataverse" thread here: https://groups.google.com/g/dataverse-community/c/UWmLYCVmVQE/m/0Zez8T7qAAAJ

**Comment by philippconzett** [comment]
I agree that with the current design, defining the license at dataset-level as applying to metadata only would cause confusion. 

Applying data reuse licenses at file-level -- where I think they belong -- needs som redesign, which should be related to support for richer metadata at file-level, as a dataset can contain heterogeneous files, which might be difficult to describe collectively at dataset level. Data file-level metadata and license information should than be aggregated on the dataset level. I'm not sure how, but a start could be to display the file-level licenses on the dataset landing page, or if there are two many files with different licenses, a notice that reusers need to consult the license information at file-level. Probably, this information should be included in a license file that is automatically created and added to a dataset. As for the metadata licence, a start could be to make that information available in a machine-readable way, so that metadata aggregaters know what they are allowed to do with the metadata about a dataset, including file-level metadata.

I think it would be a good idea to reflect on this topic in a WG. As Dataverse is being redesigned, this might be the right time.
